src/main/java/com/alvarium/annotators/AnnotatorFactory.java

```diff
@@ -0,0 +1,18 @@
+package com.alvarium.annotators;
+
+import com.alvarium.contracts.AnnotationType;
+import com.alvarium.hash.HashType;
+import com.alvarium.sign.SignatureInfo;
+
+public class AnnotatorFactory {
+
+  public Annotator getAnnotator(AnnotationType kind, HashType hash, SignatureInfo signature) 
```

Comment:
Once the issue #38 is implemented, the annotator factory will only require `AnnotationType` and `SdkInfo` params as an input. `HashType` and `SignatureInfo` will be encapsulated inside `SdkInfo`.